src/current/_includes/releases/v23.2/v23.2.0-alpha.6.md

```diff
@@ -32,21 +33,35 @@ Release Date: November 7, 2023
 - Fixed a bug in a method that was used by some of the [jobs](../v23.2/show-jobs.html) observability infrastructure. This method could be triggered if a file was overwritten with a different chunking strategy. [#113290][#113290]
 - Fixed a bug where the result of [`SHOW CREATE TABLE`](../v23.2/show-create.html) for a table that had a [collated string column](../v23.2/collate.html) with a default expression was incorrect because the statement could not be parsed. [#113119][#113119]
 - Fixed the SQL activity update job to: avoid conflicts on update, reduce the amount of data cached to only what the overview page requires, and fix the correctness of the top queries. [#112865][#112865]
+- Fixed a bug that could prevent phyiscal replication from advancing in the face of some range deletion operations. [#113041][#113041]
+- Fixed a bug where [`ALTER TYPE`](../v23.2/alter-type.html) could get stuck if [`DROP TYPE`](../v23.2/drop-type.html) was executed concurrently. [#113644][#113644]
+- Fixed a bug that could cause internal errors or panics while attempting to forecast [statistics](../v23.2/show-statistics.html) on a numeric column. [#113797][#113797]
+- Rolled back deletes no longer cause a discrepancy between computed statistics and the actual stored values. [#113766][#113766]
+
+<h3 id="v23-2-0-alpha-7-performance-improvements">Performance improvements</h3>
+
+- Addressed a performance regression that can happen when the declarative [schema changer](../v23.2/online-schema-changes.html#declarative-schema-changer) is used to create an index with a concurrent workload. [#113725][#113725]
 
 <h3 id="v23-2-0-alpha-6-contributors">Contributors</h3>
 
-This release includes 70 merged PRs by 28 authors.
+This release includes 117 merged PRs by 49 authors.
```

Comment:
Just added the merged PR total from the original and the file generated and shared in slack (@mdlinville) ... hopefully that is the correct way to go about this.